src/plot_api/edit_types.js

```diff
@@ -115,7 +116,7 @@ function falseObj(keys) {
  * @return {object} a new attributes object with `editType` modified as directed
  */
 function overrideAll(attrs, editTypeOverride, overrideContainers) {
-    var out = extendFlat({}, attrs);
+    var out = extendDeepAll({}, attrs);
```

Comment:
Using `extendDeepAll` instead of `extendFlat` because the latter throws away keys with undefined values.

Reply:
Outdated, marking as resolved.